### PR TITLE
Add tests for wrong authorization, improve current tests

### PR DESCRIPTION
**Pull request by Wuzado**
* Formatting and typo fixes in tests.py

* Add tests for wrong authorization, improve current tests

### CODE REVIEW

**Comment by Wuzado** [comment]
Squashed the commits from #19 to a single commit to reduce bloat.

File: api_server/tests.py [hunk 1]
```diff
@@ -4,53 +4,89 @@
...
+from api_server.models import Point, TypeOfPoint
 from api_server.serializers import PointSerializerDetail, TypeOfPointSerializer, PointSerializer
 
 #-------Authorization-------
 def Auth(self):
-        SECRET_KEY_TOKEN_FILE = open("./api_project/secret_key_token.txt", "r")
-        SECRET_KEY_TOKEN = SECRET_KEY_TOKEN_FILE.read()
-        SECRET_KEY_TOKEN_FILE.close()
-        token = jwt.encode({'key': 'value'}, SECRET_KEY_TOKEN, algorithm='HS256')
+    SECRET_KEY_TOKEN_FILE = open("./api_project/secret_key_token.txt", "r")
```
Comment (Vakme) [inline api_server/tests.py]: In the future: you can agree on unified code formatting

File: api_server/tests.py [hunk 2]
```diff
@@ -4,53 +4,89 @@
...
         self.token = Auth(self)
 
     def testPostType(self):
         url = '/api/points/types/add/'
         response = self.client.post(url, self.dataType, HTTP_AUTHORIZATION=self.token)
 
-        self.assertEqual(response.status_code, self.requreStatus)
-
+        self.assertEqual(response.status_code, self.requiredStatus)
+        self.assertEqual(TypeOfPoint.objects.count(), 2) # Setting it to 2 is necessary due to test DB reset before tests. (Initial type is required for the point creation)
```
Comment (Vakme) [inline api_server/tests.py]: you should check if one of these point types is the one you add

File: api_server/tests.py [hunk 3]
```diff
@@ -4,53 +4,89 @@
...
 
     def testPostPoint(self):
         url = '/api/points/add/'
         response = self.client.post(url, self.dataPoint, HTTP_AUTHORIZATION=self.token)
 
-        self.assertEqual(response.status_code, self.requreStatus)
+        self.assertEqual(response.status_code, self.requiredStatus)
+        self.assertEqual(Point.objects.count(), 1)
```
Comment (Vakme) [inline api_server/tests.py]: Same as above

**Comment by jaanonim** [comment]
if we are doing tests for 403 respons, I think we should also skip tests for 404 respons.

**Comment by jaanonim** [comment]
If we are making tests for 403 respons, I think we should also skip tests for 404 respons.

File: api_server/tests.py (hunk 1 above)
Comment (Wuzado) [reply api_server/tests.py]: Black is a decent formatter for Python.

If we're talking about this particular case, it's a clear violation of pep8 (double identation level).

File: api_server/tests.py (hunk 2 above)
Comment (Wuzado) [reply api_server/tests.py]: Leaving this for @jaanonim. He'll work on it on the current staging (mait) branch.

**Comment by Wuzado** [comment]
Merging on the request of @jaanonim.

**Pull request merged**